### ISSUE TITLE
Add testimonials from folks we gave KubeCon scholarship, they made the Tweets

### ISSUE THREAD

**Issue by Chandraprakash-Darji** [WeMakeDevs/wemakedevs#632]

**Comment by hemantwasthere** [comment]
Would love to work on this issue :)

**Comment by hemantwasthere** [comment]
Hey @Chandraprakash-Darji ,Can you please tell me where are the tweets of those students whom we gave KubeCon scholarship

**Comment by Chandraprakash-Darji** [comment]
@hemantwasthere I will ask for the information and tell you

**Comment by hemantwasthere** [comment]
Hey @Chandraprakash-Darji, Did you got the information

**Comment by Chandraprakash-Darji** [comment]
@AnuragThePathak can you help out with this?

**Comment by sambhavgupta0705** [comment]
Hey @Chandraprakash-Darji ,I would like to help you out with this issue

**Comment by AnuragThePathak** [comment]
Hey @sambhavgupta0705 would you like to share a little how you plan to work?

**Comment by sambhavgupta0705** [comment]
Hey @AnuragThePathak 
We can make a card displayed on the website which depicts the experience they shared in their tweet

**Comment by AnuragThePathak** [comment]
@Chandraprakash-Darji are you able to find the testimonials?

**Comment by Chandraprakash-Darji** [comment]
@AnuragThePathak I reached out to them, but not responded to my DM, you should try

**Comment by AnuragThePathak** [comment]
I think they didn't share any tweet most probably. BTW do you think if that blog will work.

@ashwinexe if you are active, you may be able to help us perhaps.

**Comment by ashwinexe** [comment]
> I think they didn't share any tweet most probably. BTW do you think if that blog will work.
> 
> @ashwinexe if you are active, you may be able to help us perhaps.

Hey there @AnuragThePathak happy to help! 
For context: I passed on the WMD scholarship after I got travel fund from CNCF directly. But I'm happy to provide my experience.

PS: I have a blog in my drafts about my experience in KubeConNA, waiting to be published when I get some time off.

**Comment by ashwinexe** [comment]
> @ashwinexe would you be able to share who did you pass the scholarship and do you know the other recipients?

@AnuragThePathak I do have information on that, probably the Linux foundation team or Kunal might have an idea.

**Comment by hemantwasthere** [comment]
@Chandraprakash-Darji @AnuragThePathak  Have we got any information about the tweets and what's the next plan?

**Comment by AnuragThePathak** [comment]
It will require some new additions since not all testimonials are in tweet. So probably the team members will solve it.

**Comment by Susmita-Dey** [comment]
Certainly! Here are three testimonials based on the provided posts:

1. Testimonial from Tarun Chawla's KubeCon Experience:
"Attending KubeCon was an incredible experience! The event was well-organized, packed with insightful sessions, and provided a great platform for networking with industry experts. I gained valuable knowledge and got inspired to explore new horizons in the world of Kubernetes. KubeCon truly exceeded my expectations!"

2. Testimonial from an Instagram post by user @username:
"I had the privilege of attending an amazing event recently. The energy and enthusiasm at KubeCon were contagious! The sessions were informative, covering a wide range of topics related to Kubernetes and cloud-native technologies. I left feeling motivated and excited to implement what I learned. Can't wait for the next edition of KubeCon!"

3. Testimonial from a tweet by @ashwinexe:
"KubeCon was an absolute game-changer! The event showcased the latest advancements in the Kubernetes ecosystem and offered unparalleled networking opportunities. The talks by industry leaders were insightful and thought-provoking. I left KubeCon feeling inspired and armed with new ideas to revolutionize my projects. Highly recommended!"
